src/app/organizations/collection/collection.component.ts

```diff
@@ -87,6 +91,13 @@ export class CollectionComponent implements OnInit {
     this.collectionsService.updateCollectionFromName(organizationName, collectionName);
     this.isAdmin$ = this.userQuery.isAdmin$;
     this.isCurator$ = this.userQuery.isCurator$;
+    this.organizationsService.getCollectionByName(organizationName, collectionName).subscribe((openAPICollection) => {
+      this.openAPICollection = openAPICollection;
+    });
+  }
+
+  checkHasOnlyNotebooks(collection: Collection): boolean {
+    return collection.entries.length === this.openAPICollection.notebooksLength;
```

Comment:
This use of both swagger and openapi versions of `Collection` is needed due to this [issue](https://ucsc-cgl.atlassian.net/browse/DOCK-1890). In short, `entries` is only available in the swagger `Collection` and `notebooksLength` is only available in the openapi version of `Collection`.